public/static/locales/en/sharing.json

```diff
@@ -3,7 +3,7 @@
     "dataGroupSharingDisabled": "Data and/or analyses resources cannot be shared with groups at this time",
     "fetchPermissionsError": "Failed to fetch resource permissions",
     "fetchUserInfoError": "Failed to fetch user information",
-    "noUsersFound": "No users found",
+    "noUsersFound": "No user(s) found. Enter the full CyVerse username or email of the user you wish to share.",
```

Comment:
🤔 You mentioned this in the PR description and now I'm wondering if having this as some text above the search would be nicer.  This looks good for now though, maybe we can wait to see how any additional feedback goes.